kgitb/rules/__init__.py

```diff
@@ -38,13 +38,19 @@ def _get_rules(module):
 _status_rules = _get_rules(status_rules)
 
 
+def _is_merge(status_line):
+    return status_line.startswith("Merge pull request #")
+
+
 def split_lines(commit_message):
     return re.split('\r?\n', commit_message)
 
 
 def apply(commit_message):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Redefining built-in 'apply' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=Pylint%3AW0622)